src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/DisabledExternalCredsDAO.scala

```diff
@@ -7,32 +7,42 @@ import scala.concurrent.Future
 
 class DisabledExternalCredsDAO extends ExternalCredsDAO with LazyLogging {
 
-  override def getLinkedAccount(implicit userInfo: UserInfo): Future[Option[LinkedEraAccount]] = Future.successful {
+  override def getLinkedAccount(userInfo: UserInfo): Future[Option[LinkedEraAccount]] = Future.successful {
```

Comment:
For my own edification, what does changing these from implicit to not implicit accomplish?

Reply:
Code clarity: as implicit, the user info passed in could be omitted at the function call site leaving it difficult to understand a) that credentials are actually being passed in and b) which credentials are being used. There are few good uses for implicits and this certainly is not one of them.